modules/scanner-requirements.adoc

```diff
@@ -6,19 +6,42 @@
 = Prerequisites for installing Scanner
 
 {product-title} includes an image vulnerability scanner called Scanner.
-This service scans images that are not already scanned by scanners integrated into image registries.
+This service scans images that are not already scanned by scanners integrated into image registries. Scanner also stores information about vulneribilities and scanned images in Scanner DB.
```

Comment:
```suggestion
This service scans images that are not already scanned into image registries. The Scanner stores information about vulnerabilities and scanned images in Scanner DB.
```
- The change for vulnerabilities is required, it was a misspelling. 
- I changed the sentences a little bit for easier reading. Let me know if this changes the meaning of the sentences.